src/button-split.scss

```diff
@@ -32,13 +32,13 @@ $block: #{$fd-namespace}-button-split;
     :first-child {
       border-top-left-radius: 0;
       border-bottom-left-radius: 0;
-      border-top-right-radius: $fd-border-radius;
-      border-bottom-right-radius: $fd-border-radius;
+      border-top-right-radius: var(--sapButton_BorderCornerRadius);
+      border-bottom-right-radius: var(--sapButton_BorderCornerRadius);
     }
 
     :nth-child(2) {
-      border-top-left-radius: $fd-border-radius;
-      border-bottom-left-radius: $fd-border-radius;
+      border-top-left-radius: var(--sapButton_BorderCornerRadius);
+      border-bottom-left-radius: var(--sapButton_BorderCornerRadius);
       border-top-right-radius: 0;
       border-bottom-right-radius: 0;
       border-left-width: 1px;
```

Comment:
This could also be a variable. Technically should be the same as border-width (--sapButton_BorderWidth)

Reply:
done